src/app/(core)/v/[slug]/player.tsx

```diff
@@ -132,14 +136,30 @@ export const VodPlayer = (props: { id: string; vod: VOD }) => {
           >
             {`Offset`}
           </label>
-          <TextInput
+        <div className="flex gap-2">
+        <TextInput
             type="number"
-            value={offset}
+            value={offset.seconds}
             onChange={(e) => {
-              setOffset(parseInt(e.currentTarget.value, 10));
+              setOffset(prev => {
+                const value: number = parseInt(e.target?.value, 10);
+                return {...prev, minutes: !value && value !== 0 ? "" : value, totalSeconds: (typeof value === 'number' ? value * 60 : 0) + (typeof prev.seconds === 'number' ? prev.seconds : 0)}
+              });
             }}
             suffixEl={<span className="text-gray-300">{`seconds`}</span>}
           />
+                  <TextInput
```

Comment:
This should come before the seconds field if we do it this way. 

I think I would prefer a single input that takes either seconds *or*  hh:mm:ss though, and it converts to seconds. See discussion in #36